### PR TITLE
Introduce `appsettings.{network}.json`s to easily connect to target network.

### PR DESCRIPTION
**Pull request by U-lis**
I created several `appsettings.{network].json`s to esasily connect to multiple network.
To avoid auto-connect from default value(`appsettings.json`), `appsettings.json` is now a skeleton to make another `appsettings.{network}.json`s.

- appsettings.json
  - Skeleton
  - Most of values are empty
- appsettings.mainnet.json
- appsettings.internal.json
- appsettings.previewnet.json
- appsettings.local.json
  - This file itself is ignored by `.gitignore`
  - But file entry is added to `NineChronicles.Headless.Executable.csproj` to avoid build error.

### CODE REVIEW

File: NineChronicles.Headless.Executable/appsettings.internal.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            {
+                "Name": "ByExcluding",
+                "Args": {
+                    "expression": "SourceContext = 'Libplanet.Stun.TurnClient'"
+                }
+            }
+        ]
+    },
+    "Headless": {
+        "AppProtocolVersionString": "1132/54684Ac4ee5B933e72144C4968BEa26056880d71/MEQCIHNl6d9J+118qp3gH0gGlZmuSo15a2CM8wtW+.eAglJTAiBoVWGkbXimRwft31VFpHL5qaqT3r752nLjK22PRJb5ng==/ZHU4OmxhdW5jaGVydTQyOjEvNDgwNmExOTNkNjBhMjlhYjI5ODgzNTAwZjJhNTY3Y2Q0MjUzOWY4YnU2OnBsYXllcnU0MjoxLzBmNmVjZDIwMGNhMzBjZTczY2M2ZjQ3MTc5ZjExMzQ2YzQ5MDhmMmF1OTp0aW1lc3RhbXB1MTA6MjAyMi0xMS0yNWU=",
```
[the remark this answers is not in the document]
Comment (area363) [reply NineChronicles.Headless.Executable/appsettings.internal.json:93]: I see I see. I'm not sure if this is the right way to do it but maybe we could say refer to https://github.com/planetarium/9c-k8s-config for the latest APV value in the `README.md`. What do you think about this? 🤔

[the remark this answers is not in the document]
Comment (U-lis) [reply NineChronicles.Headless.Executable/appsettings.internal.json:93]: > I see I see. I'm not sure if this is the right way to do it but maybe we could say refer to https://github.com/planetarium/9c-k8s-config for the latest APV value in the `README.md`. What do you think about this? thinking

I think that's a good idea but 9c-k8s-config is a private repo, so only members of planetarium can access to that repo.
How about providing our release config like [https://release.nine-chronicles.com/9c-launcher-config.json](https://release.nine-chronicles.com/9c-launcher-config.json)?

[the remark this answers is not in the document]
Comment (area363) [reply NineChronicles.Headless.Executable/appsettings.internal.json:93]: https://github.com/planetarium/9c-k8s-config is actually a public repo but I think https://release.nine-chronicles.com/9c-launcher-config.json is a better option! 👍

**Review by area363 — APPROVED** [review]
LGTM! Could you rebase the PR & merge? 🙏

**Pull request merged**